telegram-bot/main.go

```diff
@@ -35,25 +42,78 @@ func irc(ID int64) {
 	bot.Send(tbot.NewMessage(ID, "Join us on IRC server of Freenode at #jiit-lug. To get started refer our IRC wiki- https://github.com/osdc/community-committee/wiki/IRC ."))
 }
 
+//scraping xkcd strip URL from its website with the help of a random generated integer and then sending it as a photo using NewPhotoShare Telegram API method.
+func xkcd(ID int64) {
+	rand.Seed(time.Now().UnixNano())
+	min := 100
+	max := 2000
+	randomnum := (rand.Intn(max-min+1) + min)
+	xkcdurl := "https://xkcd.com/" + strconv.Itoa(randomnum)
+	resp, err := soup.Get(xkcdurl)
```

Comment:
@vaibhavk As you can see, firstly I am generating a random number here b/w 100 & 2000. Then I am concatenating it with `https://xkcd.com/` url since the xkcd comics have url like `https://xkcd.com/2000`. 

But what if there is no comic at any number? For that case, I have added a fullurl to one default url at line 63. Please check whether its correct or not. I have not tested that yet. Thanks.